### ISSUE TITLE
[Accessibility] Product Collection Carousel buttons rely on sensory characteristics

### ISSUE THREAD

**Issue by amberhinds** [woothemes/woocommerce#61318]
### Prerequisites

- [x] I have carried out troubleshooting steps and I believe I have found a bug.
- [x] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.

### Describe the bug

The previous/next buttons in the Product Collection Carousel rely on location to describe their purpose: "scroll products right" and "scroll products left." This description may not make sense to users without sight who don't see the carousel or that it's a scroll left/right.

Image of the buttons being read with NVDA screen reader:
<img width="1434" height="697" alt="Image" src="https://github.com/user-attachments/assets/eb5f65c0-e8e8-4768-bb4a-d24e576005ab" />

### Expected behavior

Content should not rely on sensory characteristics or location to identify them or their function. See [Failure of Success Criterion 1.3.3 due to identifying content only by its shape or location](https://www.w3.org/WAI/WCAG22/Techniques/failures/F14) 

Ideally, these buttons should follow the typical naming patterns for carousel buttons: "Previous products" and "Next Products".

This is a low-priority issue, but it would be good to standardize these names.

### Actual behavior

Current code:
```
<div data-block-name="woocommerce/product-gallery-large-image-next-previous" data-layout="{&quot;type&quot;:&quot;flex&quot;,&quot;flexWrap&quot;:&quot;nowrap&quot;}" class="wc-block-next-previous-buttons is-nowrap is-layout-flex wp-container-woocommerce-product-gallery-large-image-next-previous-is-layout-3821abfe wp-block-product-gallery-large-image-next-previous-is-layout-flex" data-wp-interactive="woocommerce/product-collection" data-wp-bind--hidden="context.hideNextPreviousButtons">
			<button aria-disabled="false" aria-label="Scroll products left" class="wc-block-next-previous-buttons__button " style="" data-wp-on--click="actions.onClickPrevious" data-wp-on--keydown="actions.onKeyDownPrevious" data-wp-bind--aria-disabled="context.isDisabledPrevious" data-wp-bind--aria-label="context.ariaLabelPrevious">
				<svg class="wc-block-next-previous-buttons__icon wc-block-next-previous-buttons__icon--left" xmlns="http://www.w3.org/2000/svg" width="8" height="12" fill="none">
					<path fill="currentColor" fillrule="evenodd" d="M6.445 12.005.986 6 6.445-.005l1.11 1.01L3.014 6l4.54 4.995-1.109 1.01Z" cliprule="evenodd"></path>
				</svg>
			</button>
			<button aria-disabled="false" aria-label="Scroll products right" class="wc-block-next-previous-buttons__button " style="" data-wp-on--click="actions.onClickNext" data-wp-on--keydown="actions.onKeyDownNext" data-wp-bind--aria-disabled="context.isDisabledNext" data-wp-bind--aria-label="context.ariaLabelNext">
				<svg class="wc-block-next-previous-buttons__icon wc-block-next-previous-buttons__icon--right" xmlns="http://www.w3.org/2000/svg" width="8" height="12" fill="none">
					<path fill="currentColor" fillrule="evenodd" d="M1.555-.004 7.014 6l-5.459 6.005-1.11-1.01L4.986 6 .446 1.005l1.109-1.01Z" cliprule="evenodd"></path>
				</svg>
			</button>
		</div>
```

### Steps to reproduce

1. Go to https://woo.equalizedigital.com/product-collection-carousel/ (Password: wooa11y)
2. Inspect the previous and next buttons and see the aria-label for the buttons.

To test on your own site:

1. Create a new post or page.
2. Add the Product Collection block and choose “create your own” so there are lots of products to display.
3. Change the layout to Carousel.
4. Set “Products in Carousel” to some higher number, e.g. 20.
5. Save and go to the frontend.
6. Inspect the previous and next buttons and see the aria-label for the buttons.

### WordPress Environment

<details><summary>System Status Report</summary>

```
### WordPress Environment ###

WordPress address (URL): [Redacted]
Site address (URL): [Redacted]
WC Version: 10.3.0-dev
Legacy REST API Package Version: The Legacy REST API plugin is not installed on this site.
Action Scheduler Version: ✔ 3.9.3
Log Directory Writable: ✔
WP Version: 6.8.3
WP Multisite: –
WP Memory Limit: 512 MB
WP Debug Mode: –
WP Cron: ✔
Language: en_US
External object cache: –

### Server Environment ###

Server Info: Apache
Server Architecture: Linux 4.19.286-203.ELK.el7.x86_64 x86_64
PHP Version: 8.1.28
PHP Post Max Size: 516 MB
PHP Time Limit: 60
PHP Max Input Vars: 1000
cURL Version: 8.7.1
OpenSSL/1.1.1w

SUHOSIN Installed: –
MySQL Version: 5.7.23-23
Max Upload Size: 512 MB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Database ###

[REDACTED]

### Post Type Counts ###

attachment: 64
aw_workflow: 5
page: 15
post: 2
product: 67
product_variation: 163
revision: 49
shop_coupon: 5
shop_order_placehold: 124
wp_global_styles: 2
wp_navigation: 1
wp_template: 2

### Security ###

Secure connection (HTTPS): ✔
Hide errors from visitors: ✔

### Active Plugins (27) ###

AutomateWoo - Birthdays Add-on: by WooCommerce – 1.3.51
AutomateWoo - Refer A Friend Add-on: by WooCommerce – 2.8.9
AutomateWoo: by WooCommerce – 6.1.16
The Bluehost Plugin: by Bluehost – 4.6.3
Disable Emails: by WebAware – 1.8.2
Password Protected: by Password Protected – 2.7.11
PU2 DevTools: by  –
User Switching: by John Blackbourn – 1.10.0
WooCommerce Smooth Generator: by Automattic – 1.2.0
WooCommerce All Products for Subscriptions: by Woo – 6.0.6
WooCommerce Back In Stock Notifications: by Woo – 3.3.101-dev
WooCommerce Composite Products: by Woo – 10.7.020-dev
WooCommerce Conditional Shipping and Payments: by Woo – 2.2.0
WooCommerce Deposits: by WooCommerce – 2.4.0
WooCommerce EU VAT Number: by WooCommerce – 3.0.0
WooCommerce Gift Cards: by Woo – 2.7.100-dev
WooCommerce Min/Max Quantities: by Woo – 5.2.5
WooPayments: by WooCommerce – 10.0.0
WooCommerce Pre-Orders: by WooCommerce – 2.3.0
WooCommerce Product Add-Ons: by WooCommerce – 8.0.101-dev
WooCommerce Product Bundles: by Woo – 8.5.01-dev
WooCommerce Square: by WooCommerce – 5.1.0
WooCommerce Subscription Downloads: by WooCommerce – 1.4.1
WooCommerce Subscriptions Gifting: by WooCommerce – 2.9.0
WooCommerce Subscriptions: by WooCommerce – 7.6.0
WooCommerce: by Automattic – 10.3.0-dev
WP Mail Logging: by WP Mail Logging Team – 1.15.0

### Inactive Plugins (6) ###

Akismet Anti-spam: Spam Protection: by Automattic - Anti-spam Team – 5.5
Braintree for WooCommerce Payment Gateway: by WooCommerce – 3.4.1
Jetpack: by Automattic – 15.0.2
Stream: by XWP – 4.1.1
UpdraftPlus - Backup/Restore: by TeamUpdraft
DavidAnderson – 1.25.8

WooCommerce Stripe Gateway: by Stripe – 9.9.2

### Must Use Plugins (1) ###

SSO: by Garth Mortensen
Mike Hansen – 0.5


### Settings ###

Legacy API Enabled: –
Force SSL: –
Currency: USD ($)
Currency Position: left
Thousand Separator: ,
Decimal Separator: .
Number of Decimals: 2
Taxonomies: Product Types: bundle (bundle)
composite (composite)
external (external)
grouped (grouped)
simple (simple)
subscription (subscription)
variable (variable)
variable subscription (variable-subscription)

Taxonomies: Product Visibility: exclude-from-catalog (exclude-from-catalog)
exclude-from-search (exclude-from-search)
featured (featured)
outofstock (outofstock)
rated-1 (rated-1)
rated-2 (rated-2)
rated-3 (rated-3)
rated-4 (rated-4)
rated-5 (rated-5)

Connected to WooCommerce.com: ✔
Enforce Approved Product Download Directories: ✔
HPOS feature enabled: ✔
Order datastore: Automattic\WooCommerce\Internal\DataStores\Orders\OrdersTableDataStore
HPOS data sync enabled: –
Enabled Features: analytics
marketplace
order_attribution
site_visibility_badge
remote_logging
email_improvements
blueprint
point_of_sale
custom_order_tables


### Logging ###

Enabled: ✔
Handler: Automattic\WooCommerce\Internal\Admin\Logging\LogHandlerFileV2
Retention period: 30 days
Level threshold: –
Log directory size: 176 KB

### WC Pages ###

Shop base: #6 - /shop/
Cart: #7 - /cart/ -  Contains the woocommerce/cart block
Checkout: #8 - /checkout/ -  Contains the woocommerce/checkout block
My account: #9 - /my-account/ -  Contains the [woocommerce_my_account] shortcode
Terms and conditions: ❌ Page not set

### Theme ###

Name: Twenty Twenty-Five
Version: 1.3
Author URL: https://wordpress.org
Child Theme: ❌ – If you are modifying WooCommerce on a parent theme that you did not build personally we recommend using a child theme. See: How to create a child theme
Theme type: Block theme
WooCommerce Support: ❌ Not declared

### Templates ###

Overrides: /home2/amberhin/public_html/website_f5ae36ad/wp-content/plugins/woocommerce/templates/block-notices/error.php
/home2/amberhin/public_html/website_f5ae36ad/wp-content/plugins/woocommerce/templates/block-notices/notice.php version 9.5.0 is out of date. The core version is 10.2.0
/home2/amberhin/public_html/website_f5ae36ad/wp-content/plugins/woocommerce/templates/block-notices/success.php

Outdated Templates: ❌
					
					
						Learn how to update					 |
					
						
					
					
						Clear system status theme info cache

### WooPayments ###

Version: 10.0.0
Connected to WPCOM: ❌ No
Logging: Disabled

### Subscriptions ###

WCS_DEBUG: ✔ No
Subscriptions Mode: ✔ Live
Subscriptions Live URL: https://woo.equalizedigital.com
Subscriptions-core Library Version: 8.3.0
Subscription Statuses: wc-active: 1
wc-cancelled: 8
wc-on-hold: 6

WooCommerce Account Connected: ✔ Yes
Active Product Key: ❌ No

### Store Setup ###

Country / State: United States (US) — Texas

### Subscriptions by Payment Gateway ###

Direct bank transfer: wc-cancelled: 1
wc-on-hold: 4

other: wc-on-hold: 1
Square: wc-active: 1

### Payment Gateway Support ###

Direct bank transfer: products
Check payments: products
Cash on delivery: products
Square: products
card_types
payment_form
authorization
charge
charge-virtual
capture_charge
refunds
voids
customer_id
tokenization
add_payment_method
token_editor
subscriptions
subscription_suspension
subscription_cancellation
subscription_reactivation
subscription_amount_changes
subscription_date_changes
multiple_subscriptions
subscription_payment_method_change_customer
subscription_payment_method_change_admin
pre-orders

Cash App Pay (Square): products
refunds
authorization
charge
charge-virtual
capture_charge


### Back In Stock ###

Database Version: 3.3.101
Loopback Test: ✔

### Composite Products ###

Database Version: ❌ 10.7.102 - Database version mismatch.
Loopback Test: ✔
Template Overrides: –

### Gift Cards ###

Database Version: 2.7.100
Loopback Test: ✔
Task Queueing Test: ✔

### Product Bundles ###

Database Version: 8.5.01
Loopback Test: ✔
Template Overrides: –

### Square ###

Environment: Production
Tokenization Enabled: ✔
Debug Mode: Save to Log

### Cash App Pay (Square) ###

Environment: Production
Debug Mode: Save to Log

### Gift Cards (Square) ###

Environment: 
Debug Mode: Save to Log

### Admin ###

Enabled Features: activity-panels
analytics
product-block-editor
coupons
core-profiler
customize-store
customer-effort-score-tracks
import-products-task
experimental-fashion-sample-products
shipping-smart-defaults
shipping-setting-tour
homescreen
marketing
mobile-app-banner
onboarding
onboarding-tasks
pattern-toolkit-full-composability
product-custom-fields
remote-inbox-notifications
remote-free-extensions
payment-gateway-suggestions
printful
shipping-label-banner
subscriptions
store-alerts
transient-notices
woo-mobile-welcome
wc-pay-promotion
wc-pay-welcome-page
launch-your-store

Disabled Features: product-data-views
experimental-blocks
experimental-iapi-mini-cart
experimental-iapi-runtime
coming-soon-newsletter-template
minified-js
product-pre-publish-modal
settings
async-product-editor-category-field
product-editor-template-system
use-wp-horizon
experimental-wc-rest-api
reactify-classic-payments-settings

Daily Cron: ✔ Next scheduled: 2025-10-08 19:10:53 -05:00
Options: ✔
Notes: 119
Onboarding: completed

### Subscriptions Gifting ###

Gifted Subscriptions Count: 4

### All Products for WooCommerce Subscriptions ###

Template Overrides: –

### Action Scheduler ###

Complete: 1,045
Oldest: 2025-10-02 17:13:53 -0500
Newest: 2025-10-07 14:58:17 -0500

Failed: 396
Oldest: 2024-09-06 00:28:33 -0500
Newest: 2025-10-07 13:07:09 -0500

Pending: 24
Oldest: 2025-10-07 14:59:50 -0500
Newest: 2025-12-08 13:49:23 -0600


### Status report information ###

Generated at: 2025-10-07 14:58:18 -05:00
```
</details>

### Isolating the problem

- [x] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [x] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/products/storefront/).
- [x] I can reproduce this bug consistently using the steps above.

**Comment by amberhinds** [comment]
@kmanijak please see above.
